kitsune/kbadge/admin.py

```diff
@@ -25,7 +19,7 @@ def show_unicode(obj):
 def show_image(obj):
     if not obj.image:
         return 'None'
-    img_url = "%s%s" % (UPLOADS_URL, obj.image)
+    img_url = "%s%s" % (settings.MEDIA_URL, obj.image)
```

Comment:
you should just be able to use the `url` property of the image field.

```suggestion
    img_url = obj.image.url
```